rnndb/bus/pci.xml

```diff
@@ -325,6 +325,14 @@
 	<stripe offset="0x78" variants="NV41-">
 		<use-group name="pci_config_exp_endpoint"/>
 	</stripe>
+	<reg32 offset="0xa8" name="CARD_LNK_CTL2" variants="G86-">
+		<bitfield low="0" high="1" name="SPEED">
+			<value value="0" name="2_5GT_v1"/>
```

Comment:
PCIe version. Even on my kepler card, if the card is in v1 mode, the value is 0. If the card is in v2 mode, the value is 1, but both mean 2.5 GT/s